README.md

```diff
@@ -174,5 +177,7 @@ Read more on [How to Write a Git Commit Message](http://chris.beams.io/posts/git
 8. When possible, one keyword to scope the change in the subject (i.e. "README: ...", "runtime: ...")
 
 
+[security]: https://github.com/opencontainers/org/blob/master/security
+[code-of-conduct]: https://github.com/opencontainers/org/blob/master/.github/CODE_OF_CONDUCT.md
```

Comment:
The link should be 
`https://github.com/opencontainers/org/blob/master/code-of-conduct.md` and  `https://github.com/opencontainers/org/blob/master/security.md`